MatrixKit/Models/Account/MXKAccountManager.m

```diff
@@ -17,6 +17,10 @@
 
 #import "MXKAccountManager.h"
 
+static NSString *const kMXKAccountsKey = @"kMXKAccountsKey";
+
+static NSString *const kMXAppGroupID = @"group.org.matrix";
```

Comment:
Would it be possible to move this key in a settings of MXKAppSettings (like MXKAppSettings.userDefaultsSuiteName) and let the app define it?

Ideally,  MXKAppSettings should have a property like ```@property  (nonatomic, readonly) NSUserDefaults *userDefaults;``` that returns ```[[NSUserDefaults alloc] initWithSuiteName:kMXAppGroupID];``` or ```[NSUserDefaults standardUserDefaults]``` if not defined (=nil).